src/parse/localization.rs

```diff
@@ -366,6 +368,7 @@ pub struct ValueParser<'a> {
     content_iters: Vec<Peekable<Chars<'a>>>,
     content_idx: usize,
     value: Vec<LocaValue>,
+    text: Cob,
```

Comment:
Can the text values be local to each method instead?
The reason is that several of the methods call each other and I'm worried you might at some point get nested usage of text.
It also seems better practice to keep them local instead of making the compiler worry about other methods maybe changing the text.